### PR TITLE
Changes to decide which co2 maps to produce (total, free, dissolved)

### PR DESCRIPTION
**Pull request by jorgesicachanr**

### CODE REVIEW

File: src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py [hunk 1]
```diff
@@ -529,33 +531,58 @@ def translate_co2data_to_property(
...
+        
         ## -999 or 0 for cells without CO2?
         #result_array = np.ma.masked_array(mass_array, mask=mask)
 
         mass_total_name = "co2_mass_total--"+str(x.date)
         mass_total_prop = xtgeo.grid3d.GridProperty(ncol=grid_pf.ncol,nrow=grid_pf.nrow,nlay=grid_pf.nlay,values=mass_total_array,name=mass_total_name,date=str(x.date))
-        mass_total_prop.to_file(grid_out_dir + "/MASS_TOTAL_"+str(x.date)+".roff", fformat="roff")
-        mass_total_prop_list.append(mass_total_prop)
+        #mass_total_prop.to_file(grid_out_dir + "/MASS_TOTAL_"+str(x.date)+".roff", fformat="roff")
+        #mass_total_prop_list.append(mass_total_prop)
```
Comment (AudunSektnanNR) [inline src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py:541]: Can remove these comments now?

File: src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py [hunk 2]
```diff
@@ -529,33 +531,58 @@ def translate_co2data_to_property(
...
 
         mass_aqu_phase_name = "co2_mass_aqu_phase--"+str(x.date)
         mass_aqu_phase_prop = xtgeo.grid3d.GridProperty(ncol=grid_pf.ncol,nrow=grid_pf.nrow,nlay=grid_pf.nlay,values=mass_aqu_phase_array,name=mass_aqu_phase_name,date=str(x.date))
-        mass_aqu_phase_prop.to_file(grid_out_dir + "/MASS_AQU_PHASE_"+str(x.date)+".roff", fformat="roff")
-        mass_aqu_phase_prop_list.append(mass_aqu_phase_prop)
+        #mass_aqu_phase_prop.to_file(grid_out_dir + "/MASS_AQU_PHASE_"+str(x.date)+".roff", fformat="roff")
+        #mass_aqu_phase_prop_list.append(mass_aqu_phase_prop)
```
Comment (AudunSektnanNR) [inline src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py:546]: Can remove these comments now?

File: src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py [hunk 3]
```diff
@@ -529,33 +531,58 @@ def translate_co2data_to_property(
...
 
         mass_gas_phase_name = "co2_mass_gas_phase--"+str(x.date)
         mass_gas_phase_prop = xtgeo.grid3d.GridProperty(ncol=grid_pf.ncol,nrow=grid_pf.nrow,nlay=grid_pf.nlay,values=mass_gas_phase_array,name=mass_gas_phase_name,date=str(x.date))
-        mass_aqu_phase_prop.to_file(grid_out_dir + "/MASS_GAS_PHASE_"+str(x.date)+".roff", fformat="roff")
-        mass_gas_phase_prop_list.append(mass_gas_phase_prop)
+        #mass_aqu_phase_prop.to_file(grid_out_dir + "/MASS_GAS_PHASE_"+str(x.date)+".roff", fformat="roff")
+        #mass_gas_phase_prop_list.append(mass_gas_phase_prop)
```
Comment (AudunSektnanNR) [inline src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py:551]: Can remove these comments now?

File: src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py [hunk 4]
```diff
@@ -529,33 +531,58 @@ def translate_co2data_to_property(
...
 
         
         print("")
-        print(type(mass_total))
-        print(f"sum of co2 mass: {mass.sum()}")
+        print(f"sum of co2 mass: {mass_total.sum()}")
         print(f"sum of co2 mass aqueous phase: {mass_aqu_phase.sum()}")
         print(f"sum of co2 mass gaseous phase: {mass_gas_phase.sum()}")        
         print(len(mass_total))
```
Comment (AudunSektnanNR) [inline src/xtgeoapp_grd3dmaps/aggregate/_co2_mass.py:558]: Maybe remove all these print statements, only for testing

File: src/xtgeoapp_grd3dmaps/aggregate/grid3d_co2_mass.py [hunk 5]
```diff
@@ -70,9 +70,9 @@ def calculate_mass_property(
 
     # POINT 4:
```
Comment (AudunSektnanNR) [inline src/xtgeoapp_grd3dmaps/aggregate/grid3d_co2_mass.py:71]: Can remove all these comments (POINT 4 etc)

**Review by AudunSektnanNR — COMMENTED** [review]
Looks good, can remove some old comments and prints.

**Comment by jorgesicachanr** [comment]
The code is clean now. Should we merge and close this PR?

**Review by AudunSektnanNR — APPROVED** [review]
Great!

**Pull request merged**